### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- New permissions

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -18,10 +18,9 @@ app=$YNH_APP_INSTANCE_NAME
 
 domain=$(ynh_app_setting_get --app=$app --key=domain)
 path_url=$(ynh_app_setting_get --app=$app --key=path)
-is_public=$(ynh_app_setting_get --app=$app --key=is_public)
 db_name=$(ynh_app_setting_get --app=$app --key=db_name)
 db_user=$db_name
-db_pwd=$(ynh_app_setting_get --app=$app --key=db_pwd)
+db_pwd=$(ynh_app_setting_get --app=$app --key=psqlpwd)
```
Comment (alexAubin) [inline scripts/upgrade:23]: Probably need to add a small migration snippet ? 

```bash
      if [ -z "$db_pwd" ]; then
      	db_pwd=$(ynh_app_setting_get --app=$app --key=db_pwd)
    	ynh_app_setting_set --app=$app --key=db_pwd
    	ynh_app_setting_set --app=$app --key=psqlpwd --value=$db_pwd
    fi
```

**Review by alexAubin — APPROVED** [review]
LGTM apart from backward compat migration snippet

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by alexAubin** [comment]
!testme

**Pull request merged**